File: Api.Server/src/Services/Center/Login/Dto/TenantLoginInput.cs
namespace Fast.Center.Service.Login.Dto;

/// <summary>
/// <see cref="TenantLoginInput"/> 租户登录输入
/// </summary>
public class TenantLoginInput
{
    /// <summary>
    /// 账号Key
    /// </summary>
    public string AccountKey { get; set; }

    /// <summary>
    /// 用户Key
    /// </summary>
    [StringRequired(ErrorMessage = "用户Key不能为空")]
    public string UserKey { get; set; }

    /// <summary>
    /// 密码
    /// </summary>
    public string Password { get; set; }
}

File: App.Admin/src/api/services/Auth/login/models/TenantLoginInput.ts
/**
 * 租户登录输入
 */
export interface TenantLoginInput {
	/**
	 * 账号Key
	 */
	accountKey?: string;
	/**
	 * 用户Key
	 */
	userKey?: string;
	/**
	 * 原始密码；生产环境必须通过 HTTPS 传输
	 */
	password?: string;
}

